src/main/java/com/harrison/controllers/EventController.java

```diff
@@ -1,5 +1,8 @@
 package com.harrison.controllers;
 
+import java.math.BigInteger;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.math.BigInteger'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://ec2-54-187-222-128.us-west-2.compute.amazonaws.com:9000/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)